.travis.yml

```diff
@@ -1,2 +1,14 @@
 language: java
-script: mvn clean verify
+sudo: false
+cache:
+  directories:
+    - $HOME/.m2
+
+env:
+  - JDK=oraclejdk8
```

Comment:
The SQLFlow repo using `openjdk-8`, do we need to keep them consistent? Ref: https://github.com/sql-machine-learning/sqlflow/blob/develop/scripts/docker/install-java.bash#L18